app/models/bookable_slot.rb

```diff
@@ -22,4 +32,14 @@ def self.for_deletion(schedule_ids)
     where(schedule_id: schedule_ids)
       .where(arel_table[:date].gteq(Time.zone.now))
   end
+
+  private
+
+  def start_at
+    Time.zone.parse("#{date} #{start[0..1]}:#{start[2..3]}")
```

Comment:
Elsewhere I've used `start.insert(2, ':')` which is perhaps a little more concise?